stylesheets/layout/layout.css

```diff
@@ -52,6 +53,7 @@
   display: none;
   z-index: 2;
   box-shadow: 1px 1px 10px 3px rgba(36, 35, 35, 0.1);
```

Comment:
you can define this in variables

Reply:
actually the box shadows were different at all places so I decided not to use variables for that, next time onwards I will try to maintain consistency and use variables, thanks